base/intfuncs.jl

```diff
@@ -841,10 +841,10 @@ end
 """
     hastypemax(T::Type) -> Bool
 
-Return `true` if and only if `typemax(T)` is defined.
+Return true if and only if the extrema `typemax(T)` and `typemin(T)` are defined.
 """
 hastypemax(::Base.BitIntegerType) = true
-hastypemax(::Type{T}) where {T} = applicable(typemax, T)
+hastypemax(::Type{T}) where {T} = applicable(typemax, T) && applicable(typemin, T)
```

Comment:
Is this change worth running PkgEval? Seems potentially mildly disruptive.

Reply:
I don't think so. But we'll see tomorrow if I'm wrong haha, and it'll be quick to revert / fix then.